firefox-kde/PKGBUILD

```diff
@@ -2,93 +2,72 @@
 # Contributor: Weng Xuetian <wengxt@gmail.com>
 # Contributor: Thaodan <theodorstormgrade@gmail.com>
 
+# enable this if you run out of memory during linking
+#_lowmem=true
+
+# enable gtk3 (warning: flash or any other plugin  crashes frequently)
+#_gtk3=true
+
 # try to build with PGO
 _pgo=true
 
-pkgname=firefox-kde
-pkgver=43.0
+_pkgname=firefox
+pkgname=$_pkgname-kde-opensuse
+pkgver=43.0.3
 pkgrel=1
-pkgdesc="Standalone web browser from mozilla.org with openSUSE's KDE integratation patches."
+pkgdesc="Standalone web browser from mozilla.org with OpenSUSE patch, integrate better with KDE"
```

Comment:
@lots0logs : Can you check the build server? I think it's stuck in "Processing packages..."